modules/auxiliary/scanner/ftp/ftp_version.rb

```diff
@@ -18,31 +18,24 @@ def initialize
       'Author'      => 'hdm',
       'License'     => MSF_LICENSE
     )
-
```

Comment:
This module was in need of some refactoring, so I did it at the same time.  In this case, the appropriate `RPORT` support already comes from `Msf::Exploit::Remote::Ftp`, so I removed it.  

In general I realize that PRs with both refactoring and features/bug fixes are not preferable, but in this case both of the affected modules are simple enough that splitting the refactoring and feature implementations into two PRs and just slows the process down.  IMO, anyway :).

Anyway, now that I've thought about it, I may actually just take the module updates out, especially in light of https://github.com/rapid7/metasploit_data_models/pull/133, and simply add rspec coverage for the mixin (which I should add regardless).